docs/Compiler_Hardening_Guides/Compiler-Options-Hardening-Guide-for-C-and-C++.md

```diff
@@ -89,6 +89,7 @@ Table 2: Recommended compiler options that enable run-time protection mechanisms
 | Compiler Flag                                                                             |            Supported by            | Description                                                                                  |
 |:----------------------------------------------------------------------------------------- |:----------------------------------:|:-------------------------------------------------------------------------------------------- |
 | [`-D_FORTIFY_SOURCE=2`](#-D_FORTIFY_SOURCE=2) <br/>(requires `-O1` or higher)             |      GCC 4.0<br/>Clang 5.0.0       | Fortify sources with compile- and run-time checks for unsafe libc usage and buffer overflows |
+| [`-D__GLIBCXX_ASSERTIONS`](#-_GLIBCXX_ASSERTIONS)                                         |      GCC ?<br/>Clang ?             | (C++ only) Run-time bounds checking for C++ strings and containers; can impact performance.  |
```

Comment:
The macro has a single preceding underscore: `_GLIBCXX_ASSERTIONS`.

Reply:
Thank you!! It was right in some places but not others. Fixed.